Buildings/Resources/Scripts/BuildingsPy/conf.json

```diff
@@ -106,5 +106,14 @@
          "rtol": 1E-8
     },
     "model_name": "Buildings.Examples.DualFanDualDuct.ClosedLoop"
+  },
+  {
+    "jmodelica": {
+      "simulate": false
+    },
+    "optimica": {
+      "simulate": false
+    },
+    "model_name": "Buildings.Applications.DHC.Loads.Controls.Validation.MixingValveControl"
```

Comment:
It may also need to be updated.

Reply:
@JayHuLBL The hyperlinks errors were already in `issue1437_district_heating_cooling`. I am surprised the CI testing did not catch them. What command did you use to detect them?

Reply:
@JayHuLBL Regarding `conf.json` I am surprised that the model now simulates with OCT and JModelica. It might be related to the changes that @mwetter did in b4cfb33. I included back the model in the CI testing with these tools.